src/FxCop/Desktop.Analyzers/Core/Security/DoNotUseInsecureCryptographicAlgorithmsAnalyzer.cs

```diff
@@ -117,6 +118,28 @@ public void AnalyzeNode(SyntaxNodeAnalysisContext context)
             {
                 SyntaxNode node = context.Node;
                 SemanticModel model = context.SemanticModel;
+                string owningParentName = string.Empty;
+                SyntaxNode cur = node;
+
+                while(cur.Parent != null)
```

Comment:
Depending on the semantics you want here either semanticModel.GetEnclosingSymbol() or semanticModel.GetContainingSymbol should probably work for you.